### PR TITLE
git-blame: add page

### PR DESCRIPTION
**Pull request by igorshubovych**

### CODE REVIEW

File: pages/common/git-blame.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# git blame
+
+> Show what revision and author last modified each line of a file
+
+- Print file with author and revision per line
```
Comment (waldyrious) [inline pages/common/git-blame.md]: How about "author and revision annotations on each line"?

File: pages/common/git-blame.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+`git blame {{file}}`
+
+- Print file with email and revision per line
```
Comment (waldyrious) [inline pages/common/git-blame.md]: Perhaps this one could say "author email" and the one above "author name", for consistency and clarity.

Comment (ostera) [inline pages/common/git-blame.md]: `commit hash` is more commonly used

**Comment by igorshubovych** [comment]
I updated it.

**Comment by waldyrious** [comment]
Nice :+1:

**Pull request merged**